auth/oidc.go

```diff
@@ -322,7 +325,7 @@ func (op *OIDCProvider) initOIDCClient() error {
 		return pkgerrors.Wrap(err, ErrMsgUnableToDiscoverConfig)
 	}
 
-	verifier = op.generateVerifier(&metadata, context.Background())
+	verifier = op.generateVerifier(&metadata, GetOIDCClientContext(op.InsecureSkipVerify))
```

Comment:
Is this fixing an unrelated issue?  If so, what's the severity of that issue?

Reply:
This is change is to fix the signature verification issue in oidc test provider while enabling unsupported option. This is a minor issue.

> What does 'in here' refer to in the PR description (and in the issue, for that matter)?

Actually, I could've used auth/oidc.go instead.